dev/deps/spark-deps-hadoop-2-hive-2.3

```diff
@@ -206,6 +208,7 @@ netty-common/4.1.77.Final//netty-common-4.1.77.Final.jar
 netty-handler/4.1.77.Final//netty-handler-4.1.77.Final.jar
 netty-resolver/4.1.77.Final//netty-resolver-4.1.77.Final.jar
 netty-tcnative-classes/2.0.52.Final//netty-tcnative-classes-2.0.52.Final.jar
```

Comment:
What is the difference between `netty-tcnative-classes` and `netty-tcnative`? netty 4.1.77 requires `netty-tcnative-classes 2.0.52.Final`